Unreal/CarlaUnreal/Config/DefaultEngine.ini

```
@@ -162,7 +163,7 @@ SoundCueCookQualityIndex=-1
 [/Script/AndroidFileServerEditor.AndroidFileServerRuntimeSettings]
 bEnablePlugin=True
 bAllowNetworkConnection=True
-SecurityToken=9054E0B944FB8898FCDC94BAD5B596A0
+SecurityToken=FCA90BC646B1F23D92A54AA8CEF4E075
```

```suggestion
SecurityToken=9054E0B944FB8898FCDC94BAD5B596A0
```

Cleanup undesired change.